db/migrate/20170717084208_add_subject_to_open_scap_result.rb

```diff
@@ -0,0 +1,6 @@
+class AddSubjectToOpenScapResult < ActiveRecord::Migration[5.0]
+  def change
+    add_column :openscap_results, :subject_id,   :bigint
+    add_column :openscap_results, :subject_type, :string
```

Comment:
If this is a polymorphic reference, prefer :resource_id, :resource_type for consistency with other models.